prov/rxd/src/rxd_ep.c

```diff
@@ -1011,8 +1011,11 @@ void rxd_ep_progress(struct util_ep *util_ep)
 	}
 
 out:
-	while (ep->posted_bufs < ep->rx_size && !ret)
+	while (ep->posted_bufs < ep->rx_size) {
 		ret = rxd_ep_post_buf(ep);
+		if (ret)
```

Comment:
Note that if there was some sort of more severe error that occurred, the progress function doesn't handle it (even to print out a message).

Reply:
Are you talking about in general or specifically with posting the buffers? rxd_ep_post_buf prints a message if there was an error.

Reply:
I was talking general to this function.  ret is being overwritten here, so any severe error has been lost.  Note that I'm not requesting changes to this patch, but future possible enhancements.